### PR TITLE
merge DR16 to main

### PR DESCRIPTION
**Pull request by Pandabear1125**
yay

### CODE REVIEW

File: src/sensors/dr16.hpp [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+    float get_l_stick_x();
+
+    float get_l_stick_y();
+
+    float get_wheel();
+
+    float get_l_switch();
+
+    float get_r_switch();
```
Comment (StudentKyle) [inline src/sensors/dr16.hpp:96]: You have undocumented members

File: src/sensors/dr16.hpp [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+/// @brief Wrapper for reading and mapping input from the DR16-Receiver and associated controller
+class DR16
+{
+public:
+	/// @brief Constructor, left empty
+	DR16();
+
+	/// @brief Initializes DR16 receiver, starts the Serial interface, and zeros input buffers
+	void Init();
```
Comment (StudentKyle) [inline src/sensors/dr16.hpp:70]: Should these functions be capitalized?

File: src/sensors/dr16.hpp [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+	/// @brief Attempts to read a full packet from the receiver. This function shouldn't be ran more than 100kHz
+	void Read();
```
Comment (StudentKyle) [inline src/sensors/dr16.hpp:73]: Capitalization?

**Review by StudentKyle — CHANGES_REQUESTED** [review]
See code notes

**Comment by StudentKyle** [comment]
Changes fixed above comments, looks good!

**Comment by Pandabear1125** [comment]
Need to add timeout for invalid data to write 0s.

**Comment by Pandabear1125** [comment]
Ready to merge, but see issue #13 for future changes

**Review by StudentKyle — APPROVED** [review]
Looks good!

**Pull request merged**